.changeset/silent-games-joke.md

```diff
@@ -0,0 +1,2 @@
+---
+---
```

Comment:
❓ @nikosdouvlis Should this changeset be empty? This change throws a more specific error message for our users. Should we mention it in our changelog? 🤔

Reply:
@anagstef i added a small description here